frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -68,14 +68,13 @@ function OverViewTab(): JSX.Element {
             width: 100,
             render: function RenderActive(_, featureFlag: FeatureFlagType) {
                 return (
-                    <LemonSwitch
-                        id={`feature-flag-${featureFlag.id}-switch`}
-                        checked={featureFlag.active}
-                        onChange={(active) =>
-                            featureFlag.id ? updateFeatureFlag({ id: featureFlag.id, payload: { active } }) : null
-                        }
-                        label={<span className="font-normal">{featureFlag.active ? 'Enabled' : 'Disabled'}</span>}
-                    />
+                    <>
+                        {featureFlag.active ? (
+                            <LemonTag type="success">Enabled</LemonTag>
+                        ) : (
+                            <LemonTag type="default">Disabled</LemonTag>
+                        )}
+                    </>
```

Comment:
I wonder what's the motivation behind replacing the interactive switch with a static tag?

Reply:
> This doesn't look right-aligned - the values should all share the same right edge / offset. The header for the column should also be right aligned with the values in the column.
> 
> <img alt="Screen Shot 2022-09-01 at 2 02 47 PM" width="1234" src="https://user-images.githubusercontent.com/254612/188012574-2905d590-182c-4908-beed-558e3ab95786.png">

The "created by" column is a reuseable column component, if I make the change it'll be all across the board, did you want that or did you just want it to be for feature flags specifically?

Reply:
For more context we replace it with a more actions dropdown option of enabling/disabling the flag but this one is for @clarkus

Reply:
That action has moved to an inline menu. You can see the full designs at https://www.figma.com/file/eX6AKgqX2bZDxgEg2S4xzy/Feature-Flags-%26-Experiments?node-id=0%3A1